config.toml

```diff
@@ -50,7 +50,7 @@ latest = "v1.11"
 fullversion = "v1.11.0"
 version = "v1.11"
 githubbranch = "master"
-docsbranch = "release-1.11"
+docsbranch = "master"
```

Comment:
what does this mean?

Reply:
i didn't add that. the only commit i did was https://github.com/kubernetes/website/pull/8617/commits/1c08038b2114d4114a190d63607790cef1103d2a

there seems to be some rebase action going on here.

Reply:
@mistyhacks i will rebase locally and only push my commit again.
+ add the changes that @tengqm requested.